### ISSUE TITLE
Empty views in the media picker

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#2414]
Instead of having a blank tab with a loading indicator when the picker is loading new site images/videos: show an empty view saying "Fetching images/videos..." (like in Post/Page screen).

If the app fails to load the media library: display a toast (already done) and change the empty view (from "Fetching" to "Unable to fetch images" - copy the Post/Page screen behavior).

**Comment by tonyr59h** [comment]
Most of the work is done. Going to wait on #2434 merge before opening PR.

**Comment by tonyr59h** [comment]
Fixed in #2499, closing.
